### PR TITLE
[CDE-849] Export of table with paginate server side set as true is only exporting 10 results in a kettle data source

### PR DESCRIPTION
**Pull request by Advent51**
fixed

### CODE REVIEW

**Comment by pamval** [comment]
@jvelasques @afrjorge please review

File: core-js/src/main/javascript/cdf/queries/CdaQuery.js [hunk 1]
```diff
@@ -212,6 +212,12 @@ define([
         queryDefinition.settingcolumnHeaders = options.columnHeaders;
       }
 
+      // only relevant for a component with server side pagination
```
Comment (afrjorge) [inline core-js/src/main/javascript/cdf/queries/CdaQuery.js]: please add the new `options.exportPage` parameter to the doc (e.g. in line 194)

File: core-js/src/main/javascript/cdf-legacy/queries/coreQueries.js [hunk 2]
```diff
@@ -564,6 +564,12 @@
         queryDefinition.settingcolumnHeaders = options.columnHeaders;
       }
 
+      // only relevant for a component with server side pagination
+      if(options.exportPage !== true) { // only export a page explicitly
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf-legacy/queries/coreQueries.js]: in order to preserve the default in all cases, change this condition to `=== false`
The comment can be updated accordingly `// only export the entire dataset when exportPage is false`

**Comment by Advent51** [comment]
@afrjorge please review

**Pull request merged**